pkg/metrics/data/fs.go

```diff
@@ -44,6 +45,13 @@ var (
 		[]string{imageRefLabel},
 		ttl.DefaultTTL,
 	)
+	TotalHangIO = prometheus.NewCounterVec(
+		prometheus.CounterOpts{
+			Name: "nydusd_hang_IO_counts",
+			Help: "Total number of hang IOs.",
```

Comment:
Please replace `hang`  with `hung` across the PR

Reply:
fixed.